plugins/inputs/zipkin/cmd/thrift_serialize/thrift_serialize.go

```diff
@@ -32,7 +32,7 @@ import (
 	"log"
 
 	"github.com/apache/thrift/lib/go/thrift"
-	"github.com/openzipkin/zipkin-go-opentracing/_thrift/gen-go/zipkincore"
+	"github.com/openzipkin/zipkin-go-opentracing/thrift/gen-go/zipkincore"
```

Comment:
Why is this necessary?

Reply:
it required downloading and installing thrift library.the url with "_thrift" did not work , checked on internet and the url with "thrift" worked. hence i had to make this change.